python/covid_graphs/covid_graphs/country_graph.py

```diff
@@ -126,8 +126,10 @@ def _create_visibility_vector(event_name: str) -> List[bool]:
         return dict(
             active=len(steps) - 1,
             currentvalue={"prefix": "Prediction date: "},
-            pad={"t": 60},
+            pad={"b": 10},
```

Comment:
If the screenshort you've attached is up2date I'd add greater bottom padding.

Reply:
Bumped it to 20.